### PR TITLE
[ADD] event_registration_mass_mailing

### PR DESCRIPTION
**Pull request by JavierIniesta**
This module adds actions to send mass mailings and allows to add to mailing
lists from event registrations tree view.

### CODE REVIEW

**Comment by JavierIniesta** [comment]
Related to OCA/social/pull/24

**Comment by JavierIniesta** [comment]
cc @dreispt @janetrp @rafaelbn @pedrobaeza

File: event_registration_mass_mailing/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+This module adds actions to send mass mailings and allows to add to mailing
+lists from event registrations tree view.
+
+Usage
+=====
+
+Event registrations tree view:
+
+* Add 'Mass mailing to partners' button: with this button, you can send an
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/README.rst]: Explain what it the menu for this

File: event_registration_mass_mailing/README.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  email to the selected registrations.
+
+* Add 'Add to mailing list' button: with this button, you can add the selected
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/README.rst]: The same

File: event_registration_mass_mailing/README.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Mass mailing from events
+========================
+
+This module adds actions to send mass mailings and allows to add to mailing
+lists from event registrations tree view.
+
+Usage
+=====
+
+Event registrations tree view:
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/README.rst]: You can remove this text

**Comment by JavierIniesta** [comment]
[IMP] README updated

**Comment by janetrp** [comment]
tested in runbot :+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by JMODaERP** [comment]
Tested in runbot :+1:

**Pull request merged**